Source/inv.cpp

```diff
@@ -869,7 +869,7 @@ void CheckInvPaste(int pnum, int mx, int my)
 			yo = PANEL_TOP;
 		}
 
-		if (i >= InvRect[r].X + xo && i < InvRect[r].X + xo + INV_SLOT_SIZE_PX) {
+		if (i >= InvRect[r].X + xo && i <= InvRect[r].X + xo + INV_SLOT_SIZE_PX) {
 			if (j >= InvRect[r].Y + yo - INV_SLOT_SIZE_PX - 1 && j < InvRect[r].Y + yo) {

```

Comment:
Should the same be done for the Y coordinate?

Reply:
I couldn't reproduce the same problem after applying the fix for X. I could do a few more tests if we want to be sure.

Reply:
im gussing its already offset via -1